test/overlay.html

```diff
@@ -347,6 +360,35 @@
         });
       });
     });
+
+    describe('frontmost', () => {
+      var dialog;
+      beforeEach(() => {
+        dialog = fixture('front-most');
+      });
+
+      it('should bring to frontmost on mousedown', (done) => {
```

Comment:
The description should better describe the issue. "should not end up behind the dialog overlay"

Reply:
done